build/jade.js

```diff
@@ -5,6 +5,9 @@ var find = require('find')
 var jade = require('jade')
 var async = require('async')
 var mkdirp = require('mkdirp')
+var extend = require('extend')
+var requireDir = require('require-dir')
+var helpers = requireDir('../helpers')
```

Comment:
Sorry those are separate points in the same comment.

I was worried about the relative path being affected by `process.cwd()`, but looking into it the require-dir module [handles it](https://github.com/aseemk/requireDir/blob/master/index.js#L12) so don't worry about that second comment.

We should handle the directory not existing though.